site/content/en/docs/Installation/Creating Cluster/eks.md

```diff
@@ -41,6 +41,12 @@ For Agones to work correctly, we need to allow UDP traffic to pass through to ou
 * Select **Inbound Rules**
 * **Edit Rules** to add a new **Custom UDP Rule** with a 7000-8000 port range and an appropriate **Source** CIDR range (`0.0.0.0/0` allows all traffic)
 
+## Use with custom CNI plugins
```

Comment:
We publish docs immediately, so we can please wrap this in a `{{% feature ... %}}` shortcode for the next release version.

See: https://agones.dev/site/docs/contribute/#at-the-page-level

Reply:
done